routes/web.php

```diff
@@ -43,5 +43,4 @@
 
 Route::get('/view', [ViewController::class, 'view'])->name('view')->middleware(['redirect-legacy-id']);
 Route::get('/demo', [DemoController::class, 'js']);
-Route::get('/frame', [FrameController::class, 'init'])->name('frame')->middleware(['migration:complete']);
-
+Route::redirect('/frame', '/#frame')->name('frame')->middleware(['migration:complete']);
```

Comment:
This absolute redirect prevents the frame mode from working for me (I don't have Lychee installed at the root URL). I'll push a fix for this (and a related issue on the landing page).

Reply:
I assume this conversation has become obsolete due to https://github.com/LycheeOrg/Lychee/commit/f3f2f6c9ea82c46a27598fc622ce57cd440add68 and https://github.com/LycheeOrg/Lychee-front/pull/323, see https://github.com/LycheeOrg/Lychee/pull/1522#discussion_r1002749404.